files/zh-cn/web/api/abortcontroller/index.html

```diff
@@ -38,11 +38,11 @@ <h2 id="方法">方法</h2>
 
 <h2 id="示例">示例</h2>
 
-<p><font>在下面的代码片段中，我们想通过</font> <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API">Fetch API</a> 下载一段视频。</p>
+<p>在下面的代码片段中，我们想通过 <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API">Fetch API</a> 下载一段视频。</p>
 
-<p><font>我们先使用</font> {{domxref("AbortController.AbortController","AbortController()")}} 构造函数<font>创建一个控制器，然后使用</font> {{domxref("AbortController.signal")}} 属性<font>获取其关联</font> {{domxref("AbortSignal")}} 对象的引用。</p>
+<p>我们先使用 {{domxref("AbortController.AbortController","AbortController()")}} 构造函数创建一个控制器，然后使用 {{domxref("AbortController.signal")}} 属性获取其关联 {{domxref("AbortSignal")}} 对象的引用。</p>
 
-<p>当一个 <a href="https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch">fetch request</a> 初始化，我们把 <code>AbortSignal</code> 作为一个选项传递到到请求对象（如下 <code>{ signal }</code>）。这将 <code>signal<code> 和 <code>controller<code> 与这个 <code>fetch request</code> 相关联，然后允许我们通过调用 {{domxref("AbortController.abort()")}} 中止请求，如下第二个事件监听函数。</p>
+<p>当一个 <a href="https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch">fetch request</a> 初始化，我们把 <code>AbortSignal</code> 作为一个选项传递到到请求对象（如下 <code>{ signal }</code>）。这将 <code>signal</code> 和 <code>controller</code> 与这个 <code>fetch request</code> 相关联，然后允许我们通过调用 {{domxref("AbortController.abort()")}} 中止请求，如下第二个事件监听函数。</p>
```

Comment:
correct those `<code>` tag. Or the `{{Specifications}}` and `{{Compat}}` are not working.